engine/src/dskw32.cpp

```diff
@@ -1497,8 +1497,6 @@ struct MCWindowsDesktop: public MCSystemInterface, public MCWindowsSystemService
 		WORD request = MAKEWORD(1, 1);
 		WSADATA t_data;
 		WSAStartup(request, &t_data);
-
-		return true;
```

Comment:
@livecodeali any idea why this return was here? Were we deliberately skipping something below or what?

Reply:
It was put here when the legacy system stuff from srvwindows.cpp was ifdef'd out - the initialisation there just did the above and then returned. Because it's using the general windows desktop implementation, it needs to do all the additional initialisation.

Reply:
ok, thanks